Source/UserManagement/Domain/StaffUser/Registering/RegisteringCommandHandlers.cs

```diff
@@ -0,0 +1,70 @@
+using doLittle.Domain;
+using doLittle.Time;
+using doLittle.Runtime.Commands;
+
+namespace Domain.StaffUser.Registering
+{
+    public class RegisteringCommandHandlers : ICanHandleCommands
+    {
+        IAggregateRootRepositoryFor<StaffUser> _repository;
+        ISystemClock _systemClock;
+
+        public RegisteringCommandHandlers(IAggregateRootRepositoryFor<StaffUser> repository, ISystemClock systemClock)
+        {
+            _repository = repository;
+            _systemClock = systemClock;
```

Comment:
Why not datetimeoffset to handle timezones?

Reply:
Testability.  You can't verify the result of DataTimeOffset.UtcNow.  The implementation of SystemClock just delegates to that in anycase, but allows mocking.